apps/explorer/src/components/OwnerObjects/OwnerObjects.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useRpcClient } from '@mysten/core';
+import {
+    type SuiObjectResponse,
+    getObjectType,
+    getObjectOwner,
+    Coin,
+} from '@mysten/sui.js';
+import { useEffect, useState } from 'react';
+
+import PaginationContainer from '../PaginationContainer/PaginationContainer';
+import OwnedObject from './components/OwnedObject';
+
+export const OBJECTS_PER_PAGE: number = 6;
+
+function OwnerCoins({ id }: { id: string }) {
+    const [results, setResults] = useState<SuiObjectResponse[]>([]);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [isFail, setIsFail] = useState(false);
+    const [currentSlice, setCurrentSlice] = useState(1);
+    const rpc = useRpcClient();
+
+    useEffect(() => {
+        rpc.getOwnedObjects({ owner: id })
+            .then((objects) => {
+                const ids: string[] = objects.data.map(
+                    (obj) => obj.data?.objectId ?? ''
+                );
+                rpc.multiGetObjects({
```

Comment:
Rather than making two calls, you should be able to do this with one, by adding `options` to the owned objects call itself